drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -144,11 +144,20 @@ def items_without_answers():
                 item['inputOptions'] = 'inputOptions' in item
             return items
 
+        def title_with_points():
+            """
+            Build title using `display_name` and `weight` of this Drag and Drop exercise.
```

Comment:
@bradenmacdonald Good point about ungraded assignments -- this change was basically just me trying to match the screenshots in the README as closely as possible :)

I reverted the change.